tooling/reproducible/ReproducibleBuilds.md

```diff
@@ -79,17 +83,60 @@ before the comparable_patch.sh can be run.
 - For BinRepl.class : export CLASSPATH=<temurin-build>/tooling/src/java:$CLASSPATH
 - A JDK for running BinRepl java : export PATH=<jdk>/bin:$PATH
 
+##### Cygwin treacherousness
+
+- It is extremely difficult (maybe impossible) to invoke `vcvarsall.bat+cl` in cygwin directly
+- Thus, it is recommended to launch this via `cmd -c` or preferably by an executable `.bat` file such as:
+
+```bash
+ pushd "$MSVSC/BUILD/TOOLS"
```

Comment:
@judovana I think this is far too complex for "doc"... I am thinking we need a helper script that does all this for a user... called something like "SetupReproEnv.bat" and "SetupComparableEnv.bat", we then make this doc very simple....thoughts?

Reply:
I have finished the publishable script yesterday. Will make PR with it today I think

Reply:
that would be great, thanks

Reply:
Note that that script is nto that simple. Depnending on MSVS, MSVC lcoations, cygwin, architectures and so on...:(

Reply:
Here it goes:

 *  https://github.com/adoptium/temurin-build/pull/3991/files#diff-2784c99fa0623d2a010b722632d7c57648182ac4c7b6138d9a6fcb44e2fe139aR196
 * https://github.com/adoptium/temurin-build/pull/3991/files#diff-2784c99fa0623d2a010b722632d7c57648182ac4c7b6138d9a6fcb44e2fe139aR142